### DISCUSSION TITLE
Screen scaling for different gamemodes

### DISCUSSION THREAD

**Discussion by AuroraStarstorm** [ppy/osu#15596] [Ideas] [answered]
I play on Osu!Lazer on a 32"inch monitor, it's really awesome to be able to letterbox osu. so it fits without having to use peripheral vision. However when I play taiko, even this reduced screen real-estate isn't enough because of my refresh rate (75hz), the notes blur as they go across the speed (because they go so fast). It'd be really awesome to reduce the horizontal scaling of taiko seperately to make it smaller (and therefore slower). 

Thanks for reading on behalf on all big monitor owners!

**Comment by bdach** [comment]
that would make gameplay easier for you. making this not a convenience feature, but a basic game balance consideration.

read https://github.com/ppy/osu/discussions/13047 for more info.

**Reply by AuroraStarstorm** [reply]
The Game balance would stay the same because it would just be shrinking the playfield down. You can already do this by manually adjusting the screen scaling settings. This would just allow the option to have a setting for each gamemode instead of a global setting, which already exists.

**Reply by bdach** [reply]
i'm sorry, i don't understand what you meant by the following then:

> It'd be really awesome to reduce the horizontal scaling of taiko seperately to make it smaller (and therefore slower).

do you want to letterbox the playfield so that it's very small and therefore the movement in physical pixels is slower?

**Reply by AuroraStarstorm** [reply]
I guess so, but you can already do what I'm asking for. It'd just be adding 4 versions of it for each mode instead of a general option.
![image](https://user-images.githubusercontent.com/94187757/141517492-41682d7d-f7e9-499b-b37d-6d410787907a.png)
![image](https://user-images.githubusercontent.com/94187757/141517554-5b3285bf-faf7-4045-832e-386c4679d622.png)

**Comment by AuroraStarstorm** [comment]
The option for reference, just this but instead of gameplay, an option for each gamemode so you don't have to manually toggle for each one.

![image](https://user-images.githubusercontent.com/94187757/141510166-2a295380-a76d-40e2-b4e6-f2289a54aa82.png)

**Comment by smoogipoo** [comment, marked as answer]
We will likely implement gameplay settings "profiles" (I swear there was another issue/discussion somewhere), but this is quite low priority at this stage and is at least dependent on the new database implementation.
